File: android/app/src/main/java/com/igaming/html5app/plugin/LoggerBridgePlugin.java
package com.igaming.html5app.plugin;

import android.content.SharedPreferences;
import android.util.Log;

import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "LoggerBridge")
public class LoggerBridgePlugin extends Plugin {

    @PluginMethod
    public void log(PluginCall call) {
            String message = call.getString("message", "");
            if (message != null) {
                Log.d("LoggerBridge", message);
            }
            call.resolve();
    }

    @PluginMethod
    public void error(PluginCall call) {
        String message = call.getString("message", "");
        if (message != null) {
            Log.d("LoggerBridge", message);
        }
        call.resolve();
    }
}

File: ios/App/App/LoggerBridgePlugin.swift
//
//  LoggerBridgePlugin.swift
//  App
//
//  Created by KimSungjin on 3/25/26.
//

import Foundation
import Capacitor

@objc(LoggerBridge)
public class LoggerBridge: CAPPlugin {

    @objc func log(_ call: CAPPluginCall) {
        let message = call.getString("message") ?? ""
        NSLog("[LoggerBridge] %@", message)
        call.resolve()
    }

    @objc func error(_ call: CAPPluginCall) {
        let message = call.getString("message") ?? ""
        NSLog("[LoggerBridge][ERROR] %@", message)
        call.resolve()
    }
}
